src/Model.php

```diff
@@ -600,6 +661,32 @@ public function isDirty($fields = [])
         return false;
     }
 
+    /**
+     * @param string|null $filter
+     *
+     * @return array
+     */
+    public function getFields(string $filter = null)
+    {
+        if (!$filter) {
+            return $this->fields;
```

Comment:
Not sure about this, but if we don't set $filter, then it will return fields also outside of only_fields scope.
Is that OK?

Reply:
yes, that seems to be the best, since we recommend to replace foreach($this->elements) with getFields() and that would break it for only_fields then. (2 tests actually break if i apply only_fields scope).

we could add new filter 'accessible', which would respect only_fields, if needed.

Reply:
Fine.